lib/redis_memo/redis.rb

```diff
@@ -17,12 +17,10 @@ def initialize(
           if option.is_a?(Array)
             RedisMemo::Redis::WithReplicas.new(option)
           else
-            option[:logger] ||= RedisMemo::DefaultOptions.logger
```

Comment:
Why do we need to remove this? Can we just set the redis logger level to debug too?

Reply:
Well, no one really uses those logs from the redis-rb client. It's only useful to redis-memo devs. And even for us, it's we rarely use it -- it can be replaced with `redis monitor` anyway.